ansible/roles/satellite-configuration/tasks/03_content_view.yml

```diff
@@ -0,0 +1,85 @@
+---
+- name: Fetching list of names available in Content-Views
+  shell: hammer --output cve content-view list --organization "{{ORG}}" |grep Name
+  register: content_view_list
+  ignore_errors: yes
+
+- name: Creating list fact of names available in Content-Views
+  set_fact: 
+    content_view_exist: "{{ content_view_exist + [item.split(':')[1].lstrip(' ')]}}"
+  loop: "{{content_view_list.stdout_lines}}"
+
+# - name: Print fact
+#   debug: var=content_view_exist
+
+- name: Creating Content-view if not exist
+  shell: hammer content-view create --name "{{CONTENT_VIEW_NAME}}"  --organization "{{ORG}}"
+  when: CONTENT_VIEW_NAME not in content_view_exist
+  tags:
+    - satellite_configuration
+    - content-view
+#######################################
+
+- name: Fetching list of enabled Repo IDs from subscription 
+  shell:  'hammer --output cve  repository list --organization "{{ORG}}" |grep Id'
+  ignore_errors: yes
+  register: repo_list
+  tags: 
+    - satellite_configuration
+    - content-view
+
+- name: Creating list fact of enabled Repo IDs from subscription 
+  set_fact:
+    sub_repos_name_exist: "{{ sub_repos_name_exist + [ item.split(':')[1].lstrip(' ')] }}"
+  loop: "{{repo_list.stdout_lines}}"
```

Comment:
What happens to this task and the following tasks if `Fetching list of enabled Repo IDs from subscription` fails ?
(`ignore_errors`)

Reply:
Fixed